sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -1774,7 +1776,7 @@ SUBROUTINE CALVOR(UWND,VWND,ABSV)
 !     
 !     LOOP TO COMPUTE ABSOLUTE VORTICITY FROM WINDS.
 !     
-      IF(MODELNAME  == 'RAPR' .AND. SUBMODELNAME /= 'MPAS') then ! for RAP / HRRR only
+      IF(MODELNAME  == 'RAPR') then
 !$omp  parallel do private(i,j)
```

Comment:
@KarinaAsmar-NOAA May I know the reason of this modification?

Reply:
@WenMeng-NOAA I think those modifications happened by mistake during a merge. I'll remove them.